shuffle-examples/pom.xml

```diff
@@ -60,13 +60,19 @@ under the License.
 
 							<archive>
 								<manifestEntries>
-									<program-class>com.alibaba.flink.shuffle.examples.BatchJobDemo</program-class>
+									<program-class>
+										com.alibaba.flink.shuffle.examples.BatchJobDemo
+									</program-class>
 								</manifestEntries>
 							</archive>
 
 							<includes>
-								<include>com/alibaba/flink/shuffle/examples/BatchJobDemo.class</include>
-								<include>com/alibaba/flink/shuffle/examples/BatchJobDemo$*.class</include>
+								<include>
```

Comment:
Why are these three changes in `pom.xml` in diff? I don't get the point.

Reply:
These changes are made by IDE, will revert these changes.